scico/loss.py

```diff
@@ -215,31 +217,31 @@ def prox(
             ATWA = c * A.conj() * W * A
             return lhs / (ATWA + 1.0)
 
-        #   prox_{f}(v) = arg min  1/2 || v - x ||^2 + λ α || A x - y ||^2_W
+        #   prox_{f}(v) = arg min  1/2 || v - x ||_2^2 + λ 𝛼 || A x - y ||^2_W
         #                    x
         # solution at:
         #
-        #   (I + λ 2α A^T W A) x = v + λ 2α A^T W y
+        #   (I + λ 2𝛼 A^T W A) x = v + λ 2𝛼 A^T W y
```

Comment:
I don't think extends beyond characters in a comment. What are you recommending here?

Reply:
Put another way: It looks like you are proposing to change "α" to "𝛼".
Question 1) Why?
Question 2) Have you done it across the entire codebase? If not, we have a mixture of subtly different alphas, that seems bad.
Implied comment: add this to the pile if reasons I'm skeptical of unicode in code.

Reply:
Only locally, and only in a comment. Does this really have implications for usage elsewhere. The font change was intended to improve visibility since a small "α" looks a bit like an "a" in a different Roman font.